docs/src/bundles.md

```diff
@@ -0,0 +1,13 @@
+# Bundles
+
+Bundles are a convenient way of adding the required components for a given entity.
+For instance, the Minecraft entity `Piglin` requires the marker `Piglin`, `Uuid`, `Position`, `Physics` and `Health`.
```

Comment:
> requires the marker `Piglin`, ... 

The way this is written almost implies that all of those components are "markers." An easy way to fix this could be 
> For instance, the Minecraft entity `Piglin` requires these components: `Uuid`, `Position`, `Physics`, and the marker component, `Piglin`